artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/ActiveMQTestBase.java

```diff
@@ -176,6 +178,15 @@
  */
 public abstract class ActiveMQTestBase extends Assert {
 
+   // MBeanServer is an expensive object.
+   // Even if you called unregisterMBeanServer it will still be allocated somewhere in the VM.
+   // this will make the testsuite a lot lighter
+   static MBeanServer mBeanServer = MBeanServerFactory.createMBeanServer();
```

Comment:
If all the tests are largely going to use the same MBeanServer, for all their brokers, I'd wonder if they couldn't just use the probably-already-existing _PlatformMBeanServer_ and remove need for this?
https://docs.oracle.com/javase/8/docs/api/java/lang/management/ManagementFactory.html#getPlatformMBeanServer--